R/utils.R

```diff
@@ -365,6 +365,39 @@ build_comm_cards <- function(data){
             is_perc = 0,
             is_url = 0)
   
+  if (time_diff_first_rel$value < 2) {
+    return(cards)
+  }
+  
+  trend_downloads <- dplyr::as_tibble(data) %>% 
+    dplyr::arrange(year, month) %>%
+    dplyr::filter(row_number() >= (n() - 11)) %>% 
+    dplyr::mutate(row_n = row_number())
+  
+  model_result <- lm(downloads ~ row_n, data = trend_downloads) %>% 
+    broom::tidy() %>% 
+    dplyr::filter(term == "row_n") %>% 
+    dplyr::mutate(estimate = round(estimate, 0)) %>% 
+    dplyr::mutate(succ_icon = dplyr::case_when(
+      estimate > 0 ~ "arrow-trend-up",
+      estimate < 0 ~ "arrow-trend-down",
+      TRUE ~ "bars"
+      )
+    ) %>%
+    dplyr::select(estimate, succ_icon)
+  
+  cards <- cards %>%
+    dplyr::add_row(
+      name = 'downloads_trend',
+      title = 'Downloads trend',
+      desc = 'Trend of downloads in last 12 months',
```

Comment:
Technically, the trend data could be fit on anywhere from 12 to 24 months worth of data, correct? I think it'd be easier to just leave the time period out of the description.  If you want to take the easy route, I think this description more accurately conveys the analysis, no?

```suggestion
      desc = 'Trend in average downloads per month',
```
Or "Monthly trend in downloads, on average" would work too. 

Or you could insert a dynamic time period- one that changes from package to package. For example a newer package may be fit to 14 months of data if the first release was 14 months ago, but `stringr` would always be fit to 24 months of data since it's been around a long time.

Reply:
I really liked the idea of showing the number of months being considered for the analysis, it's ranging from 12-24 months and I explicitly state it in the card now